### ISSUE TITLE
[Fixed in GH-6052] safari firefox: Missing Visual Focus on Add-To-Cart Modal

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4317]
Visual Focus on Closing Button and Increase/Decrease Qty are missing, as well as regular buttons.

![Screen Shot 2019-08-29 at 2 13 32 PM](https://user-images.githubusercontent.com/15113219/63967324-2f933a00-ca6b-11e9-8ac6-9fff4df7dc21.png)

**Comment by kacperknapik** [comment]
- Chrome: ✔
- Firefox: ✔ (note: focus border is huge)
- Safari: ❓

**Comment by KateChuen** [comment]
Chrome: OK

Missing: 
- the border on the whole qty rectangle isn't as obvious on firefox
<img width="738" alt="Screen Shot 2019-10-28 at 3 37 11 PM" src="https://user-images.githubusercontent.com/45409515/67711923-5ac9d500-f999-11e9-8009-dda78b351b03.png">


- the border around the `-` sign is cut in firefox, like mentioned in #4319
- view cart & . proceed to checkout buttons on firefox & safari
- focus on the `- and +` signs on safari
- remove button on safari

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-30 at 4 09 43 PM](https://user-images.githubusercontent.com/15113219/73490689-06a90500-437b-11ea-8b6a-2d244bea00b9.png)
![Screen Shot 2020-01-30 at 4 11 15 PM](https://user-images.githubusercontent.com/15113219/73490752-29d3b480-437b-11ea-88ef-1ecd738dac31.png)
